From b9a4154942c932a493736d2292bbae16384e35df Mon Sep 17 00:00:00 2001
From: Federico Cuello <fedux@fedux.com.ar>
Date: Sat, 17 Nov 2018 17:44:49 +0100
Subject: [PATCH] Fix ypart begin for bpf ranges

It was creating yparts with begin=0 and end==previous_part_begin with
multipart files. It should start from 1 and next part from previous end
+ 1.
---
 uulib/uuencode.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/uulib/uuencode.c b/uulib/uuencode.c
index b9a390e..2768771 100644
--- a/uulib/uuencode.c
+++ b/uulib/uuencode.c
@@ -1740,7 +1740,7 @@ UUEncodePartial_byFSize (FILE *outfile, FILE *infile,
             }
 
           fprintf (outfile, "=ypart begin=%ld end=%ld%s",
-                   (partno-1)*bpf,
+                   (partno-1)*bpf+1,
                    (partno*bpf) < progress.totsize ?
                    (partno*bpf) : progress.totsize,
                    eolstring);
